site/es/blog/new-in-devtools-96/index.md

```diff
@@ -28,171 +28,172 @@ draft: true
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b2a04e234f25602d1b7e7ff7bd0d39bde3f2c1ec  #}
 
 Chromium issue: [1254259](https://crbug.com/1254259)
 
 
-<!-- ### Retain the Console sidebar {: #console-sidebar } -->
+### Mantener la barra lateral de la consola {: #console-sidebar }
 
-<!-- The Console sidebar is here to stay. In Chrome 94, we announced the [upcoming deprecation of the Console sidebar](/blog/new-in-devtools-94/#deprecated) and ask developers for feedback and concerns. -->
+La barra lateral de la consola volverá a quedarse. En Chrome 94, anunciamos la [desactivación de la barra lateral de la consola](/blog/new-in-devtools-94/#deprecated) y pedimos a los desarrolladores comentarios y opiniones.
 
-<!-- We have now got enough feedback from the deprecation notice and we will work on improving the sidebar rather than removing it. -->
+Ahora tenemos suficientes comentarios sobre el aviso de desactivación y trabajaremos para mejorar la barra lateral en lugar de eliminarla.
 
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XIsLjvBFSeaTN5BtEgmU.png", alt="Console sidebar", width="800", height="502" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b0650096c934bf60c21d51ae8a51c94e8f907d38 #}
 
 Chromium issues: [1232937](https://crbug.com/1232937), [1255586](https://crbug.com/1255586)
 
+## Panel de caché de la aplicación en desuso (deprecated) en el panel de la aplicación  {: #app-cache } -->
 
-<!-- ## Deprecated Application cache pane in the Application panel {: #app-cache } -->
-
-<!-- The [Application cache](https://developer.chrome.com/docs/devtools/storage/applicationcache/) pane in the Application panel is now removed as the support for [AppCache](https://web.dev/appcache-removal/) is removed from Chrome and other Chromium-based browsers. -->
+El panel [Caché de la aplicación](https://developer.chrome.com/docs/devtools/storage/applicationcache/) en el panel de la aplicación se eliminó como el soporte para [AppCache](https://web.dev/appcache-removal/) se elimina de Chrome y otros navegadores basados ​​en Chromium.
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/de4d15e955d6145674e3885cde8a5a70f1269b79 #}
 
-Chromium issue: [1084190](https://crbug.com/1084190) 
+Chromium issue: [1084190](https://crbug.com/1084190)
 
 
-<!-- ## [Experimental] New Reporting API pane in the Application panel {: #reporting-api } -->
+## [Experimental] Nuevo panel de la API de informes en el panel de la aplicación {: #reporting-api }
 
 {% Aside %}
-<!-- To enable the experiment, check the **Enable Reporting API panel in the Application panel** checkbox under **Settings** > **Experiments**. -->
+Para habilitar esta característica experimental, marque la casilla de verificación **Habilitar el panel de API de informes en el panel de la aplicación** que encontrará en la siguiente ruta **Configuración** > **Experimentos**.
```

Comment:
I think the string with the experiment name is not translated. Ignore this comment if I'm wrong.
```suggestion
Para habilitar esta característica experimental, marque la casilla de verificación **Enable Reporting API panel in the Application panel** que encontrará en la siguiente ruta **Configuración** > **Experimentos**.
```

Reply:
You're right, experimental panel is still untranslated.